static/widgets/libs-widget.ts

```diff
@@ -296,6 +375,44 @@ export class LibsWidget {
         }
     }
 
+    async getBuildInfoAsHtml(libId: string, semver: string, url?: string): Promise<string> {
+        const details = await lib_annotations.getForCompiler(libId, semver, this.currentCompilerId);
+
+        let libInfoText = '';
+        for (const info of details) {
+            if (info.annotation.commithash) {
+                const machineName = shortenMachineName(info.annotation.machine || '');
+                const stdlib = info.buildinfo.libcxx;
+                if (url && url.startsWith('https://github.com/')) {
+                    // this is a bit of a hack because we don't store the git repo in our properties files
```

Comment:
lol not _that_ much of a hack compared to so many other parts of our code, but thanks for the honest comment :)